packages/gradle/src/executors/gradlew/schema.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://json-schema.org/schema",
+  "version": 2,
+  "title": "GradlewImpl executor",
+  "description": "The GradlewImpl executor is used to run Gradle tasks.",
```

Comment:
should we mention that it's basically just runCommands unless in batch mode? Or is that too much implementation detail?

Reply:
i am going to remove the gradlew executor for now because native batch runner will change it anyway, since it is not used for now